CMakeLists.txt

```diff
@@ -35,6 +35,8 @@ option(LOVR_SYSTEM_OPENXR "Use the system-provided OpenXR" OFF)
 option(LOVR_BUILD_EXE "Build an executable (or an apk on Android)" ON)
 option(LOVR_BUILD_SHARED "Build a shared library (takes precedence over LOVR_BUILD_EXE)" OFF)
 option(LOVR_BUILD_BUNDLE "On macOS, build a .app bundle instead of a raw program" OFF)
+option(LOVR_BUILD_WITH_EXPORTED_INTERNALS "When building as a shared library, make all C symbols visible. This exposes internals, so be aware that any APIs you use from C may change at any time." OFF)
+option(LOVR_BUILD_EXTRA_PLUGINS "Any additional plugins you want to include in the build system" "")
```

Comment:
Maybe plugins could always be sourced from the top-level plugins folder instead of the one inside the lovr repo?

Reply:
@bjornbytes ah, from whatever the top CMakeLists.txt is? I like that idea!

Reply:
@bjornbytes This is implemented now :)